syntax = "proto3";

package proto_cv;

// msgHead is nesscessary. It is used as common message head for all message . 

message msgHead{
    int32 len = 1;
}

message msgMat{
    int32 rows = 1;
    int32 cols = 2;
    int32 elt_type = 3;
    int32 elt_size = 4;
    bytes mat_data = 5;
}
